Source/WebCore/svg/SVGElement.cpp

```diff
@@ -606,7 +606,7 @@ bool SVGElement::childShouldCreateRenderer(const Node& child) const
     case ElementNames::SVG::tspan:
         return false;
     default:
-        break;        
```

Comment:
Please update your IDE to not do this. This is generally frown upon in the WebKit project, though I won't ask you to revert from this patch.

Reply:
Discussions with others led me to believe this was okay. I tagged you on the relevant Slack thread.